auth/credentials/impersonate/user.go

```diff
@@ -29,6 +29,10 @@ import (
 	"cloud.google.com/go/auth/internal"
 )
 
+var (
+	iamCredentialsEndpoint = "https://iamcredentials.googleapis.com"
```

Comment:
Should this endpoint be universe aware?

Reply:
No, user authentication is not supported.